tasks/main.yml

```diff
@@ -1,10 +1,16 @@
 # file: postgresql/tasks/main.yml
 
-- include_vars: "{{ item }}"
+- name: PostgreSQL | Include Variables | OS Family
+  include_vars: "{{ item }}"
   with_first_found:
-    - "../vars/{{ ansible_os_family }}.yml"
-    - "../vars/empty.yml"
-  tags: [always]
+    - "{{ ansible_os_family }}.yml"
+
+- name: PostgreSQL | Include Variables | OS Family, OS Major Version
+  include_vars: "{{ item }}"
+  with_first_found:
+    - files:
+      - "{{ ansible_os_family }}-{{ ansible_distribution_major_version }}.yml"
+      skip: true
```

Comment:
I'm not sure if this is clear... but I didn't want to have to duplicate *ALL* of the variables from `RedHat.yml` into new files.  This format basically says:

* Load the defaults from the `{{ ansible_os_family }}.yml` file
* If a version specific file exists, then load that... which will override the default variables. If no file exists, then skip this step and just use the defaults.

This lets you put most of the variables in the main {{ ansible_os_family }}.yml file, and only put the _differences_ into the version specific file.